jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -247,6 +258,16 @@
                for tenant in api.config.tenants.tenants
                if std.objectHas(tenant, 'mTLS')
              ] else []) +
+            (if std.objectHas(api.config.tenants, 'tenants') then [
+               if std.objectHas(tenant.oidc, 'configMapName') then {
```

Comment:
you're getting nulls because this line doesn't qualify what elements to loop over but what to output for that iteration. This `if` should go after line 269.